behaviors/magic-button.md

```diff
@@ -47,4 +51,18 @@ url: $SITE_PREFIX (this is a ~ special token ~ that evaluates to the prefix of c
 property: url
 ```
 
+### (ﾉ◕ヮ◕)ﾉ*:・ﾟ✧ "grab the image url from a lede component, then ask mediaplay for the caption"
+
+```yaml
+component: feature-lede
+transform: getComponentInstance
+url: $SITE_PREFIX
+property: imgUrl
+moreMagic:
+  -
+    transform: mediaplayUrl (to change the image url into a string we can query mediaplay's api with)
+    url: [mediaplay api url]
+    property: metadata.caption
+```
```

Comment:
Could there be a `magic` property to contain all magic unless there is something special about the first magic? Something like

```yaml
component: feature-lede
magic:
  -
    transform: getComponentInstance
    url: $SITE_PREFIX
    property: imgUrl
  -
    transform: mediaplayUrl (to change the image url into a string we can query mediaplay's api with)
    url: [mediaplay api url]
    property: metadata.caption
```